### PR TITLE
added is_top_level to the cql_libraries

### PR DESCRIPTION
**Pull request by zlister**
Pull requests into cqm-execution require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1927
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name: @losborne 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name: @hackrm 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: spec/fixtures/json/patients/CMS134v6/Pass_Numer.json [hunk 1]
```diff
@@ -1,623 +0,0 @@
-{
```
Comment (losborne) [inline spec/fixtures/json/patients/CMS134v6/Pass_Numer.json]: This patient was removed because of the ucum unit issue--need to change the "cc" to "cm3" and re-export

File: spec/fixtures/json/patients/CMS32v7/Visit_1ED.json [hunk 2]
```diff
@@ -159,7 +159,7 @@
         "id": {
           "namingSystem": null,
           "qdmVersion": "5.4",
-          "value": "5c40c81657a11e474f19bb89"
+          "value": "5c8b9f30a564acdfc38acf68"
```
Comment (losborne) [inline spec/fixtures/json/patients/CMS32v7/Visit_1ED.json]: `Id` changes aren't really necessary to re-commit, but if it's tricky to get them all aligned without committing, im fine with leaving them in.

File: spec/fixtures/json/cqm_measures/CMS13v2/CMS13v2.json [hunk 3]
```diff
@@ -12958,25 +12977,30 @@
   },
   "population_sets": [
     {
-      "id": "PopulationCriteria1",
+      "population_set_id": "PopulationCriteria1",
```
Comment (losborne) [inline spec/fixtures/json/cqm_measures/CMS13v2/CMS13v2.json]: Is this change related to your update? Or did this somehow git missed in a previous commit?

File: spec/fixtures/json/cqm_measures/CMS123v7/CMS123v7.json [hunk 4]
```diff
@@ -7099,10 +7100,12 @@
           "statement_name": "Initial Population",
           "statement_references": [
             {
+              "hqmf_id": null,
```
Comment (losborne) [inline spec/fixtures/json/cqm_measures/CMS123v7/CMS123v7.json]: Are these hqmf_id changes from a previous commit?

File: spec/fixtures/json/cqm_measures/CMS13v2/CMS13v2.json (hunk 3 above)
Comment (zlister) [reply spec/fixtures/json/cqm_measures/CMS13v2/CMS13v2.json]: population_set was added in a previous commit, I guess it just didn't get committed for this file.

File: spec/fixtures/json/cqm_measures/CMS123v7/CMS123v7.json (hunk 4 above)
Comment (zlister) [reply spec/fixtures/json/cqm_measures/CMS123v7/CMS123v7.json]: I didn't touch the hqmf_id stuff in this update. The only thing that changed was adding `is_top_level` to the new fixtures
